setup.cfg

```diff
@@ -26,6 +26,8 @@ norecursedirs = .git build .tox specs .cache udata/static udata/templates udata/
 python_files = test_*.py
 python_functions = test_*
 python_classes = *Test
+env =
+    AUTHLIB_INSECURE_TRANSPORT=true
```

Comment:
What’s this?

Reply:
Because HTTPS is part of OAuth specs and `authlib` enforce it unless this environment variable is explicitly set.
This allows tests (and only tests) to run OAuth flows in http (because the test serveur is `http://localhost`).

Reply:
Ok, I missed the pytest section above.